libs/authorizenet/src/effects/authorize-net.effects.ts

```diff
@@ -66,16 +72,23 @@ export class DaffAuthorizeNetEffects<T extends DaffAuthorizeNetTokenRequest = Da
 		map((actions: Actions[]) => new DaffAuthorizeNetUpdatePaymentFailure('The payment method has failed to update the cart.'))
 	)
 	
-	@Effect({ dispatch: false })
+	@Effect()
   loadAcceptJs$ : Observable<any> = this.actions$.pipe(
     ofType(DaffAuthorizeNetActionTypes.LoadAcceptJsAction),
-		tap((action: DaffLoadAcceptJs) => {
-			const acceptJsScript = document.createElement('script');
-			acceptJsScript.async = true;
-			acceptJsScript.setAttribute('type', 'text/javascript');
-			acceptJsScript.setAttribute('src', this.config.production ? ACCEPT_JS_PRODUCTION_URL : ACCEPT_JS_SANDBOX_URL);
-			acceptJsScript.setAttribute('charset', 'utf-8');
-			document.body.appendChild(acceptJsScript);
-		})
-  )
+		map((action: DaffLoadAcceptJs) => {
+			if(typeof Accept === 'undefined') {
+				const acceptJsScript = document.createElement('script');
```

Comment:
`document` isn't available server-side. You should probably extract most of this code to a separate service and inject it here.